local-example/.kube.yml

```diff
@@ -3,38 +3,38 @@ apiVersion: extensions/v1beta1
 kind: Deployment
 
 metadata:
-  name: {{.app}}-{{.env}}
+  name: {{.app_name}}-{{.env}}
 
 spec:
   replicas: 3
   template:
     metadata:
       labels:
-        app: {{.app}}
+        app: {{.app_name}}
         env: {{.env}}
     spec:
       containers:
         - name: app
-          image: {{.image}}
+          image: {{.app_image}}
```

Comment:
and this as `{{.image}}`

Reply:
so while working with the project and documenting a few commands / parameters, i noticed this (as well as the `{{.app}}` and `{{.API_TOKEN}}` references) were potentially confusing

specifically, it seemed like folks might be easily confused between`{{.API_TOKEN}}` and `secrets: { source: GOOGLE_CREDENTIALS, target: token }` especially when looking for `token` and seeing references to both the SA credential use case and the `secretRef` use case.. same with the k8s container spec `image` use case and the drone step `image` use case, as well as in the k8s container spec `name` use and the pod spec `label` use case..  

more generally, the idea here was to try and distinguish between references related to a user's use case and references related to the plugin's use case..

mind elaborating a bit on your concerns here?